README.md

```diff
@@ -2,13 +2,13 @@
 
 ## Getting started
 
-The project uses vagrant to run in a suitably configured VM. You will need Vagrant and VirtualBox installed.
+The project uses docker to run in a suitably configured container. You will need Docker installed.
 
-Copy the src/.env.template file to a new file `src/.env`, and populate it with your trello api credentials and board ID. 
+Copy the `.env.template` file to a new file `.env`, and populate it with your trello api credentials and board ID. 
 
 Then start the Flask app by running:
 ```bash
-$ vagrant up
+$ docker-compose up
```

Comment:
You shouldn't really *require* developers to use docker - so you should have instructions to run `poetry install`/`poetry run flask run` and the alternatives of `vagrant up` / `docker-compose up`

Also, FYI this command will not rebuild the image even if you edit the docker file. Safer to use `docker-compose up --build`

Reply:
have added --build, running locally was covered further down the readme, I've added the vagrant instructions back in